.github/workflows/build.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           - linux_openresty_1_17
         test_dir:
           - t/plugin
-          - t/admin t/cli t/config-center-yaml t/control t/core t/debug t/discovery t/error_page t/misc
+          - t/admin t/cli t/config-center-yaml t/control t/core t/debug t/discovery t/error_page t/misc t/pubsub
           - t/node t/router t/script t/stream-node t/utils t/wasm t/xds-library t/xrpc
```

Comment:
plugin < pubsub < node, so we should put it here

Reply:
fixed #7043 , According to https://github.com/apache/apisix/pull/6995#discussion_r869835933, I have alphabetically aligned it between `node` and `router`